libs/dh/wholesale/feature-settlement-reports-v2/src/lib/modal/dh-request-settlement-report-modal.component.html

```diff
@@ -0,0 +1,83 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<watt-modal
+  *transloco="let t; read: 'wholesale.settlementReportsV2.requestReportModal'"
+  [title]="t('title')"
+  #modal
+>
+  <form id="request-settlement-report-form" [formGroup]="form" (ngSubmit)="submit()">
+    <vater-stack align="stretch" direction="column" gap="l">
+      <div class="items-group">
```

Comment:
Suggestion: ![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1250254/5f02f53f-878d-4065-a7a4-1d83c576c223)
The item group do not seem to matter?

Reply:
Question: Instead of the itemsgroup divs, could a with on form not do the same?

Reply:
The `div` matters in terms of `gap` on the wrapping `vater-stack`.  The `div` prevents gap between element + checkbox.

With

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/a60d6029-9283-4089-b4ed-b1fedac4cdd7)

Without

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/691f63fc-3467-4b8b-bc4a-bf604f7eb7a1)

Reply:
Aha :-)